catalog/generators/database-crud-dotnet/merge/csproj-mysql

```diff
@@ -1 +1,2 @@
     <PackageReference Include="MySql.Data.EntityFrameworkCore" Version="8.0.15"/>
+    <PackageReference Include="AspNetCore.HealthChecks.MySql" Version="2.2.0"/>
```

Comment:
@omajid we need to figure out when we need to update version numbers of these packages.

Reply:
Good point. It looks like they only did a release along with ASP.NET Core 2.2 and used the same version number. But we definitely need a strategy for updating these.